CHANGELOG.next.asciidoc

```diff
@@ -77,6 +77,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - The `elasticsearch/deprecation` fileset now indexes the `component` field under `elasticsearch` instead of `elasticsearch.server`. {pull}10445[10445]
 - Remove field `kafka.log.trace.full` from kafka.log fielset. {pull}10398[10398]
 - Change field `kafka.log.class` for kafka.log fileset from text to keyword. {pull}10398[10398]
+- Address add_kubernetes_metadata processor issue where old source field is 
+  still used for matcher. {issue}10505[10505] {pull}10506[10506]
```

Comment:
This should go under the "Affecting all Beats" header. Sorry for requesting another change on this.

Reply:
This change seems to only apply to Filebeat because it uses the source/log.file.path field. It does not affect other beats.

Reply:
Ah, cool, good to go then.